src/vmm/src/guest_config/aarch/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+
+use crate::guest_config::Modifier;
+
+/// Template providing bitmask modifier info
+/// for CPU register configuration.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
```

Comment:
super nit: keep `Deserialize` and `Serialize` derives together.

Reply:
Formatters are ordering alphabetically. Without automatic formatting here I am not going to make changes in this regard.